packages/database/test/helpers/util.ts

```diff
@@ -21,10 +21,23 @@ import '../../index';
 import { Reference } from '../../src/api/Reference';
 import { Query } from '../../src/api/Query';
 import { ConnectionTarget } from '../../src/api/test_access';
-import { RepoInfo } from '../../src/core/RepoInfo';
 
 export const TEST_PROJECT = require('../../../../config/project.json');
 
+const USE_EMULATOR = process.env.USE_RTDB_EMULATOR;
```

Comment:
You probably don't need it and can just check the port number.

Reply:
I hesitated as well. I guess code is more readable this way?

Reply:
i'd like to have as few env variables as possible. I would do const USE_EMULATOR = !! EMULATOR_PORT;

Reply:
Done.